moped-editor/src/queries/project.js

```diff
@@ -245,25 +242,29 @@ export const TIMELINE_QUERY = gql`
         subphase_id
       }
     }
-    moped_subphases(
-      where: { subphase_id: { _gt: 0 } }
-      order_by: { subphase_order: asc }
-    ) {
-      subphase_name
-      subphase_id
-    }
     moped_proj_phases(
       where: { project_id: { _eq: $projectId }, is_deleted: { _eq: false } }
-      order_by: { phase_start: desc }
+      order_by: {
+        phase_start: desc
+        moped_phase: { phase_order: desc }
+        moped_subphase: { subphase_order: desc }
+      }
```

Comment:
trying to make the row ordering consistent on the phases table between mutations - but really should be using material table sort methods on top of this.